openapi/openapi-client/src/main/openapi3/master_rest_v1.yaml

```diff
@@ -93,6 +93,58 @@ paths:
         "503":
           description: Dynamic configuration is disabled.
 
+  /api/v1/conf/dynamic/upsert:
```

Comment:
The following code block is identical to the worker's. You can refer to this link to reuse them(https://dev.to/mikeralphson/defining-reusable-components-with-the-openapi-specification-4077).